packages/venia-concept/package.json

```diff
@@ -52,6 +52,7 @@
     "@magento/peregrine": "~6.0.0",
     "@magento/pwa-buildpack": "~5.1.1",
     "@magento/upward-js": "~4.0.1",
+    "@magento/upward-security-headers": "*",
```

Comment:
This probably should follow the pattern of other deps and be `~0.0.1`.

Reply:
Fixed to this constraint in 27e45b1